solr/core/src/java/org/apache/solr/search/ReRankQParserPlugin.java

```diff
@@ -99,6 +109,7 @@ protected float combine(
   private static final class ReRankQuery extends AbstractReRankQuery {
     private final Query reRankQuery;
     private final double reRankWeight;
+    private final String reRankOperator;
 
     @Override
     public int hashCode() {
```

Comment:
minor: should we also consider the `reRankOperator` here?

Reply:
Do you mean in the `hashCode`? Probably yes - I'd not noticed that! I'll have to familiarise myself with what that's for.